### PR TITLE
SaltyBet and Cuevana3 button update 

### PR DESCRIPTION
**Pull request by magumboi**
Screenshot_27
Screenshot_26
Screenshot_31

### CODE REVIEW

File: websites/S/Salty Bet/presence.ts [hunk 1]
```diff
@@ -71,52 +141,60 @@ presence.on("UpdateData", async () => {
...
+    (isBetOpen()) ? browsingStamp = Math.floor(Date.now() / 1000) : presenceData.startTimestamp = browsingStamp;
+
+    switch(mode[0]){
+      case "trofeo":
+        presenceData.buttons = [
+          { label: "Tournament Bracket",
+            url: "https://www.saltybet.com/shaker?bracket=1"
+          },
+          { label: "Visit SaltyBet.com",
+            url: "https://www.saltybet.com"
```
Comment (ririxidev) [inline websites/S/Salty Bet/presence.ts:158]: image

File: websites/S/Salty Bet/presence.ts [hunk 2]
```diff
@@ -71,52 +141,60 @@ presence.on("UpdateData", async () => {
...
+          }
+        ];
+        break;
+      case "saltgirl":
+        presenceData.buttons = [
+          { label: "Exhibition Queue",
+            url: "https://www.saltybet.com/shaker?activerequests=1"
+          },
+          { label: "Visit SaltyBet.com",
+            url: "https://www.saltybet.com"
```
Comment (ririxidev) [inline websites/S/Salty Bet/presence.ts:168]: image

File: websites/S/Salty Bet/presence.ts [hunk 3]
```diff
@@ -71,52 +141,60 @@ presence.on("UpdateData", async () => {
...
+        presenceData.buttons = [
+          { label: "Visit SaltyBet.com",
+            url: "https://www.saltybet.com"
+          }
+        ];
+        break;
+      default:
+        presenceData.buttons = [
+          { label: "Visit SaltyBet.com",
+            url: "https://www.saltybet.com"
```
Comment (ririxidev) [inline websites/S/Salty Bet/presence.ts:182]: image

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Comment by magumboi** [comment]
image

**Review by ririxidev — APPROVED** [review]
looks fine for me 👍

**Review by Bas950 — APPROVED** [review]

**Pull request merged**